setup.py

```diff
@@ -100,6 +100,6 @@ def bake_version(v):
         ]
     },
     data_files=get_baked_files(),
-    python_requires=">=3.7",
+    python_requires=">=3.8",
```

Comment:
Should we add a notice somewhere saying that we are dropping support of python 3.7?

Reply:
I think we can put that into the release note.